src/scss/patterns/_code.scss

```diff
@@ -29,6 +29,14 @@ pre {
   white-space: pre;
 }
 
+code:not([class^='language']) {
+  background: var(--ccs-special--bg5-fade);
+  border-radius: var(--quarter-shim);
+  color: var(--ccs-accent--fg1);
```

Comment:
@mirisuzanne I wasn't sure which colors are available to use for stuff this like so if this isn't allowed, let me know.

Reply:
All the colors are available, but it might be good to assign them more semantic names before using them? For example, `var(--ccs-accent--fg1)` already has an alias as `var(--accent)` in the color config.

(and very strangely also `var(--feature-accent)`… maybe we have some cleanup to do 😅)